Resources/Help/analyses/ANCOVA.md

```diff
@@ -0,0 +1,227 @@
+- Sum of Squares 
+    - Type I: Sequential sum of squares. It is the reduction of error when each factor of the model is added to the factors already included, containing the order in the model. 
+    - Type II: Hierarchical/Partially sequential sum of squares. It is the reduction of error when a factor is added to the model that includes all the other factors, except the factors where the added factor is part of. Therefore, it does not include interactions. 
+    - Type III: Partial sum of squares. It is the reduction of error when a factors is added at last to the model, a model that includes all the other factors, including interactions with this factor. This type is selected by default. 
+
+#### Assumption Checks: 
+- Homogeneity tests: By selecting this option it will be checked if the variance of the dependent variable is equal between the groups by using Levene's test.  
+- Q-Q plot of residuals: Checks the validity of the distributional assumption of the data set. More specific, from the plot it could be retrieved if the data is normally distributed and linear. 
+
+#### Contrasts: 
+For every independent variable you can select a specific contrast by clicking on the `none` in the right column.  
+- Factors: These are the independent variables included in the analysis, so the variables placed in the `Fixed Factors` box.  
+- Contrasts: Contrast allow you to conduct planned comparisons, for example to compare the different levels of the independent variable with each other. There are different contrasts that allow for different types of comparisons.  
+    - none: By selecting this option no contrasts are calculated. This option is selected by default. 
+    - deviation: By selecting this contrast the mean of each level of the independent variable is compared with the overall mean (the mean when all the levels are taken together). 
+    - simple: When this contrast is selected the mean of each level is compared to the mean of a specified level, for example with the mean the control group. 
+    - difference: This contrast is also called reverse Helmert. By selecting this contrast the mean of each level is compared to the mean of the previous levels. 
+    - Helmert: When this contrast is selected the mean of each level is compared to the mean of the following levels. This is the reverse of the difference contrast. 
+    - repeated: By selecting this contrast the mean of each level is compared to the mean of the following level. 
+    - polynomial: This contrast test polynomial trends in the data. The specific polynomial that will be used for the analysis depends on the amount of levels of the independent variable. The degree of the trend used for the analysis is the amount of levels - 1. Therefore, if the independent variable consist of 2 levels, a linear trend is analysed. If the independent variable consists of three levels, a quadratic trend is analysed besides to linear. 
```

Comment:
tests
number of levels
minus 1 (so type out the minus)
a quadratic trend is analysed in addition to the linear trend.